app/assets/javascripts/discourse/tests/acceptance/create-account-user-fields-test.js

```diff
@@ -39,9 +38,8 @@ acceptance("Create Account - User Fields", function (needs) {
     assert.ok(exists(".user-field"), "it has at least one user field");
 
     await click(".modal-footer .btn-primary");
-    assert.ok(exists("#modal-alert"), "it shows the required field alert");
     assert.equal(
-      queryAll("#modal-alert").text(),
+      queryAll("#account-email-validation").text().trim(),
```

Comment:
why queryAll and not query ?

Reply:
No idea, was there in the test already, will change